tools/codegen/codespec/api_to_provider_spec_mapper_test.go

```diff
@@ -42,11 +42,11 @@ var (
 	}
 
 	simpleTestResourceOperations = codespec.APIOperations{
-		Create: codespec.APIOperation{
+		Create: &codespec.APIOperation{
```

Comment:
would add a unit test in which a config defines data sources configuration and we can assert the resulting model